### ISSUE TITLE
how to add scrollbars in chart?really need them.

### ISSUE THREAD

**Issue by vatsal78** [NielsLindberg/Qliksense.Extension.amWaterfall#6]

**Comment by NielsLindberg** [comment]
Hi,

I have updated the extension to include an option for adding a scrollbar to the categoryaxis.

Br.
Niels

**Comment by vatsal78** [comment]
Hi,
Thank you for scroll bar.
Is there any way i can fix the size of the bar? Even i scroll the bar size
should not change, right now it changes as soon as i scroll till end.
Thanks in advance for your future help to me.

Waiting for your reply.
Regards,
Vatsal

**Comment by NielsLindberg** [comment]
Hey Vatsal,

Is it possible for you to send a screenshot of the bug you are mentioning?

Br.
Niels

**Comment by vatsal78** [comment]
Yes sure,you can check the attachment. In the default combo chart the bar
size is always same while we scroll but in your combo chart it changes when
we scroll so is there any way to fix this?

Thanks & Regards,
Vatsal

**Comment by vatsal78** [comment]
Hey,
I got this i just didn't set the scroll size done with this, now issue i am
facing is the Y-axis value changes automatically while i am scrolling. I
want it to stay static. Even i want to keep value on data points is that
possible in this chart? In bar chart we can keep the value on the on
datapoints similar i want to have in combo chart.

Regards,
Vatsal

**Comment by NielsLindberg** [comment]
Hi vatsal,

You can set the value labels on the measure settings here:

![image](https://cloud.githubusercontent.com/assets/16624342/21094529/a865afec-c057-11e6-8ab7-ed2be36133bb.png)

Regarding static y-axis, at the moment you can only fix the minimum part of the axis here, i might add maximum aswell then the way to do a static axis over e.g. a chart with month dimension could be. max-axis = max(aggr(measure,dimension))*1.1 and min-axis = min(aggr(measure,dimension))*-1.1

![image](https://cloud.githubusercontent.com/assets/16624342/21094566/cf468ff0-c057-11e6-8f5c-8ebdcade0cdc.png)

**Comment by vatsal78** [comment]
Thank you very much.
Is there any way i can make this chart responsive? You can just give me
clue how to do it as i know little bit of coding so i can work on it and
complete it.
Regards,
Vatsal

**Comment by NielsLindberg** [comment]
Responsive, you mean change according to viewport sizes?

Hmm i think you need to maybe update the amcharts library that i'm using then import the following plugin and enable it as they demonstrate in this link https://www.amcharts.com/kbase/making-charts-responsive/

Br.
Niels

**Comment by vatsal78** [comment]
Yes i will try updating amcharts library,
I am attaching an image is there any way to reduce the space between legend
and chart i was able to reduce the size of legend but it didn't make any
change to chart view.
Regards,
Vatsal

**Comment by vatsal78** [comment]
I have updated the library but the viewport responsive is bot working, i am
attaching a screenshot of what i exactly need. The default charts are
visible when we are reducing the size of screen but in your chart it does
not stay the same ir visible do help me with this things.sorry for talking
your lot of time.
Regards,
Vatsal

**Comment by NielsLindberg** [comment]
Hey, man don't worry about it :p I appreciate the feedback and ill look into the stuff at my own pace :) when possible. Can't seem to see the screenshots, you can just copy paste pictures directly into the text areas here btw.

**Comment by vatsal78** [comment]
Yes no problem.

Thank you

**Comment by vatsal78** [comment]
Hey,
I wanted to know is it possible to have the calculated labels in this
extension?
Your help will be really appreciated.
Regards,
Vatsal Shah
